(AzureCXP) fixes MicrosoftDocs/azure-docs#77925 (#77985)
Azure/azure-content: articles/azure-functions/functions-app-settings.md

## WEBSITE\_CONTENTOVERVNET

> For all Windows virtual network-supported SKUs in the Dedicated (App Service) plan and for Windows Elastic Premium plans. Consumption and Linux Elastic Premium plans aren't supported. A value of `1` enables your function app to scale when you have your storage account restricted to a virtual network. You should enable this setting when restricting your storage account to a virtual network. To learn more, see [Restrict your storage account to a virtual network](functions-networking-options.md#restrict-your-storage-account-to-a-virtual-network).

Reviewer: We need to also fix the branding in the VNET article.
```suggestion
Supported for Windows on a Premium plan and Dedicated (App Service) plans (Standard and higher). Not supported for Linux Consumption and Premium plans. A value of `1` enables your function app to scale when you have your storage account restricted to a virtual network. You should enable this setting when restricting your storage account to a virtual network. To learn more, see [Restrict your storage account to a virtual network](functions-networking-options.md#restrict-your-storage-account-to-a-virtual-network).
```

Author: What exact changes are required here?
